django_material_demo/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: "3.9"
+   
+services:
+  db:
+    image: postgres
+    ports:
+      - 5432:5432
+    env_file:
+      - .env
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+    
+  web:
+    restart: always
```

Comment:
Don't put `restart: always` here. You will need to turn it off constantly after restarting machine if you work on other projects.